lib/providers/plate_weights.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/widgets.dart';
+import 'package:wger/helpers/gym_mode.dart';
+
+class PlateWeights extends ChangeNotifier{
+  String unitOfPlate = 'kg';
```

Comment:
I think a boolean would be better here, something like isMetric, then we don't have to use magic strings in the code, which will avoid mistakes like e.g. writing "Lb"